packages/ui-extensions/docs/surfaces/admin/staticPages/configuration.doc.ts

```diff
@@ -0,0 +1,99 @@
+import {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const data: LandingTemplateSchema = {
+  title: 'Configuration',
+  description:
+    'When you create an admin UI extension, the shopify.ui.extension.toml file is automatically generated in your admin UI extension directory. This guide describes the properties that you can configure in shopify.ui.extension.toml.',
+  image: '/assets/landing-pages/templated-apis/hero.png',
+  darkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  tabletImage: '/assets/landing-pages/templated-apis/hero.png',
+  tabletDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  mobileImage: '/assets/landing-pages/templated-apis/hero.png',
+  mobileDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  id: 'configuration',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'how-it-works',
+      title: 'How it works',
+      sectionContent: `You define properties for your admin UI extension in the extension configuration file. The shopify.extension.toml file contains the extension's configuration, which include the extension name, api_version, extension targets and modules.
+
+These extensions are currently in a Develoepr Preview so they cannot be published to Shopify.`,
+    },
+    {
+      type: 'GenericList',
+      anchorLink: 'configuration-options',
+      title: 'Configuration Options',
+      listItems: [
+        {
+          name: 'name',
+          description:
+            'The name of the extension. This is the name that will be displayed in the container of your extension.',
+          value: 'string',
+          isOptional: false,
+        },
+        {
+          name: 'api_version',
+          description:
+            'The version of the Components and APIs that the extension uses',
+          value: 'string',
+          isOptional: false,
+        },
+        {
+          name: 'extensions',
+          description:
+            'This is a grouping of properties that define the extension being developed.',
+          value: 'string',
+          isOptional: false,
+        },
+        {
+          name: 'handle',
+          description:
+            'The handle of the extension. This value is not merchant facing.',
+          value: 'string',
+          isOptional: false,
+        },
+        {
+          name: 'type',
+          description:
+            'This is an enum value indicating what the extension run time is. For all current extensions, this value will be ui_extension.',
+          value: 'ui_extension',
+          isOptional: false,
+        },
+        {
+          name: 'extensions.targeting',
+          description:
+            'This is a grouping of extension targets that the extension is targeting.',
+          value: 'string',
+          isOptional: false,
+        },
+        {
+          name: 'module',
```

Comment:
I'm not sure if it's feasible, but it would be good to structure this page in a way that makes it clear that `handle` explicitly belongs underneath `extensions.targeting` (and the same for all the other nested properties listed here). For now I think this is fine

Reply:
Yea, I agree. I think the folks working on the unified toml will have more detailed docs and I'll mimic the pattern they use for it here once that is ready/ships!

Reply:
Do we need to repeat the docs again here instead of linking to it https://github.com/Shopify/shopify-dev/pull/34566/files#diff-c23276940c260d6e77eee8a6a207e1d39c801fa802e385d0f3bc9a5b148ef2fe? These are all standard UI extensions config, nothing specific to Admin.